RFC.md

```diff
@@ -0,0 +1,466 @@
+# [RFC] Fused Models
+
+**TLDR**
+- Fused Models are two+ pre-trained models joined together and further tuned to work as one model. This is the approach used for most SOTA Multimodal models currently.
+- This RFC proposes special modules and utils similar to our PEFT implementation to allow for easy manipulation of these fused model parameters and their state dicts.
+
+**WHY**: To support a wide variety of multimodal models and multimodal recipes with a consistent single model API.
+## Context
+The majority of SOTA multimodal models coming out these days are not trained end to end as uniform models, but instead a separate model is trained for each modality and then these models are fused together. In the simplest case, you have an LLM trained on text and an image encoder such as CLIP. You then need to combine these pre-trained models and finetune them so that the LLM can understand the embeddings from the image encoder.
```

Comment:
What does "uniform models" mean in this context?

Reply:
By this I mean a single model -- all of the parameters trained together.